templates/web/base/reports/_status_filter_options.html

```diff
@@ -0,0 +1,9 @@
+    [%~ IF c.cobrand.on_map_default_status == 'open' %]
+      data-none="[% loc('Open') %]"
+    [%~ END ~%]
+    data-all="[% loc('All') %]"
+    [%~ IF c.user.is_superuser OR c.user.from_body == body ~%]
+    data-all-options='["confirmed","investigating","action scheduled","closed","fixed - council","in progress", "internal referral", "not responsible","duplicate","unable to fix"]'
```

Comment:
These states shouldn't be hard-coded any more. It needs to be the states in filter_states, easiest with a loop similar to that which prints out the `<option>`s.